File: Karaoke/Arquivo-em-Java/model/Pessoa.java
public class Pessoa {
  private static int contadorID = 1;
  private int id;
  private String nome;
  private String musica;

  public Pessoa(String nome, String musica) {
    this.id = contadorID++;
    this.nome = nome;
    this.musica = musica;
  }

  public int getID() { return id; }
  public String getNome() { return nome; }
  public String getMusica() { return Musica; }
  
  public void setMusica (String setMusica) { 
    this.musica = musica; 
  }
  
  @Override 
  public String toString() { 
    return "ID: " + id + " | Nome: " + nome + " | Música: " + musica; 
  }
}

File: Karaoke/Arquivo-em-Python/Model/pessoa.py
class Pessoa:
    __contador_id = 1

    def __init__(self, nome, musica):
        self.__id = Pessoa.__contador_id
        Pessoa.__contador_id += 1
        self.__nome = nome
        self.__musica = musica

    def get_id(self):
        return self.__id

    def get_nome(self):
        return self.__nome

    def get_musica(self):
        return self.__musica

    def set_musica(self, musica): 
        self.__musica = musica

    def __str__(self):
        return f"ID: {self.__id} | Nome: {self.__nome} | Música: {self.__musica}"